.github/workflows/release.yml

```diff
@@ -4,46 +4,197 @@ on:
     tags:
       # this is a glob, not a regexp
       - '[0-9]*'
+
+
 jobs:
-  cockpituous:
+  source:
     runs-on: ubuntu-latest
-    environment: release
     container:
-      image: ghcr.io/cockpit-project/release
+      image: ghcr.io/cockpit-project/unit-tests
+      options: --user root
     permissions:
-      # create GitHub release
       contents: write
     steps:
-      - name: Set up configuration and secrets
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Workaround for actions/checkout#697
```

Comment:
Please specify the full URL https://github.com/actions/checkout/pull/697 here (took me a short while to find this).

I wonder why we didn't stumble over this before?